lib/bolt/bolt_option_parser.rb

```diff
@@ -64,6 +64,15 @@ def get_help_text(subcommand, action = nil)
       when 'guide'
         { flags: OPTIONS[:global] + %w[format],
           banner: GUIDE_HELP }
+      when 'module'
```

Comment:
Was this intentionally left out of the help text on line 180 of this file?

Reply:
Yes. Otherwise, users would see it as an available command in the help text even though it's currently gated.